extensions/supernotes/src/components/CardDetail.tsx

```diff
@@ -0,0 +1,45 @@
+import { Action, ActionPanel, Detail, Icon } from "@raycast/api";
+
+import { useStoreCard } from "../hooks/useRecent";
+import { PermMap } from "../util/mapping";
+import { ICard } from "../util/types";
+
+const CardDetail = ({ card }: { card: ICard }) => {
+  useStoreCard(card);
+
+  return (
+    <Detail
+      navigationTitle={card.data.name}
+      markdown={card.data.markup}
+      actions={
+        <ActionPanel>
+          <Action.CopyToClipboard title="Copy HTML" content={card.data.html} icon={Icon.Clipboard} />
+          <Action.CopyToClipboard title="Copy Markdown" content={card.data.markup} icon={Icon.Clipboard} />
```

Comment:
Related to the copying: it seems like currently all text copied via the Raycast Clipboard API is done so with the mimetype `text/plain`, however for our above "Copy HTML" option it would be much better if it was copied with type `text/html` so that when pasted into other applications it doesn't just paste the raw HTML (e.g. when pasting into an email client). 

Is this something you think you could support?